### DISCUSSION TITLE
Negative planar coordinates values

### DISCUSSION THREAD

**Discussion by jaimebayes** [pyproj4/pyproj#843] [Q&A] [answered]
Hi guys, 

I am trying to understand why conversion from geographic (4326) to planar (3116) results negative values, as I know, planar avoids negative values. Any insight about it? Thanks, JL

```
from pyproj import Transformer
transformer = Transformer.from_crs(4326, 3116)
e, n = transformer.transform(lng, lat)
```
-75.527122, 10.419584 ----> -9351863.050885107, 2625791.505009844
-75.243001, 4.44466 ----> -9175145.802284697, 2631619.4996994324
-76.532494, 3.451792 ----> -9179613.003146926, 2480877.2238357333
-75.517325, 5.06807 ----> -9199232.696515208, 2604278.1389542604

**Comment by snowman2** [comment, marked as answer]
Likely related to axis order: #538

**Reply by jaimebayes** [reply]
Great, it works now!! 
Thank you very much, 

JL

**Reply by snowman2** [reply]
Glad to hear that it worked 👍
